### PR TITLE
semih week-2

### PR DESCRIPTION
**Pull request by sigma21**

### CODE REVIEW

File: week2/homework/src/index.js [hunk 1]
```diff
@@ -1,3 +1,66 @@
...
 
 // TODO: Write the homework code in this file
+
+let fs = require('fs');
+
+let args = process.argv.slice(2);
+
+let command = args[0];
+let todoItem = args[1];
+
```
Comment (eidosam) [inline week2/homework/src/index.js]: All the above variables are not re-assigned, so `const` should be used to declare them.

File: week2/homework/src/index.js [hunk 2]
```diff
@@ -1,3 +1,66 @@
...
+
+
+if(command === 'add') {
+  add(todoItem);
+} else if (command === 'help'){
+  help();
```
Comment (eidosam) [inline week2/homework/src/index.js]: Try to run `node . help`; you will get an error

    ReferenceError: helpText is not defined

So you need to move the declaration of `helpText` to before the function call.

File: week2/homework/src/index.js [hunk 3]
```diff
@@ -1,3 +1,66 @@
...
+} else if (command === 'list') {
+  list(todoItem);
+} else if (command === 'remove') {
+  remove(todoItem);
+};
+
```
Comment (eidosam) [inline week2/homework/src/index.js]: No command or invalid one should be also covered.

File: week2/homework/src/index.js [hunk 4]
```diff
@@ -1,3 +1,66 @@
...
+      console.log(error);
+    } else {
+      console.log(todoList);
+    };
+  });
+};
+
+function add(todoItem){
+  fs.appendFile('./todoList.txt', todoItem + '\n', (error) => {
+    console.error(error);
```
Comment (eidosam) [inline week2/homework/src/index.js]: You should also check the error and print it only if there is. Also better to report to user the status, like  

    if(error) {
        console.error(error);
    } else {
        console.log('Item is added.');
    }

Like what you did in `remove` function.

File: week2/homework/src/index.js [hunk 5]
```diff
@@ -1,3 +1,66 @@
...
+  });
+};
+
+function remove(todoItem){
+  fs.readFile('./todoList.txt', 'utf8', (err, data) => {
+    if (err) {
+      throw err;
+    } else {      
+      let dataArray = data.split('\n');
```
Comment (eidosam) [inline week2/homework/src/index.js]: `dataArray` is not re-assigned, use `const`

**Comment by wouterkleijn** [comment]
Thanks @eidosam for the comments. Please @sigma21  can you make sure to implement the feedback?

**Comment by wouterkleijn** [comment]
@sigma21 Any updates?

**Comment by sigma21** [comment]
@wouterkleijn @eidosam thanks for the feedback!
I've just updated the PR according to your comments.

**Comment by wouterkleijn** [comment]
Super, guys - thanks a lot :)

**Pull request closed without merging**